src/buckets/actions/thunks.ts

```diff
@@ -144,8 +145,25 @@ export const createBucketAndUpdate = (
       bucketSchema
     )
 
+    // just created the bucket, now add any schemas that might be there:
     dispatch(addBucket(newBucket))
     dispatch(checkBucketLimits())
+
+    if (resp.data.schemaType === 'explicit' && schemas && schemas.length) {
+      // in case they choose explicit, add schemas, then change their mind back and
+      // change it to implicit; or if explicit and choose not to add schemas at this time.
```

Comment:
I'm a little confused by this comment. what line does it apply to? Is it guarding against the situation described in the comment, or is it enabling the situation described in the comment?